docs/architecture/adr-066-rosetta-plugins.md

```diff
@@ -0,0 +1,121 @@
+# ADR-066: Rosetta Plugins (multi zone=
+
+## Changelog
+
+* May 30, 2023: Initial Draft: {changelog}
+
+## Status
+
+{DRAFT | PROPOSED} Not Implemented
+
+## Abstract
+
+Enabling Rosetta in multiple zones requires adding interfaces and codec types.
+Forking the project and implementing it would require a lot of effort. The proposed approach
+aims to load such interfaces and types in a simple and easy way instead of implementing
+Roseta as a whole
+
+## Context
+
+The Cosmos ecosystem contains multiple zones with different message types and interfaces.
+Rosetta is a tool that allows chains to be listed by Coinbase while also providing a standard
```

Comment:
I'm not sure you wanted comments on the english hehe, but I think this would be clearer as "while also providing standardized access to chain information"